### PR TITLE
Use the new style for configuration variables description for Plex pl…

### PR DESCRIPTION
**Pull request by oncleben31**
**Description:**
Implement the new style for configuration variables description as requested by #6385  

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by frenck** [comment]
Hi there @oncleben31!

Thank you for this PR, however, the build is failing. Could you please take a look?

File: source/_components/media_player.plex.markdown [hunk 1]
```diff
@@ -72,14 +72,48 @@ media_player:
...
-- **use_custom_entity_ids** (*Optional*): Name Entity ID's by client ID's instead of friendly names. Defaults to `false`.  HA assigns entity ID's on a first come first serve basis.  When you have identically named devices connecting (ex. media_player.plex_web_safari, media_player.plex_web_safari2), you can't reliably distinguish and or predict which device is which.  This option avoids this issue by using unique client ID's (ex. media_player.dy4hdna2drhn).
-- **use_episode_art** (*Optional*): Display TV episode art instead of TV show art. Defaults to `false`.
-- **remove_unavailable_clients** (*Optional*): Remove stale plex clients from UI after interval. Defaults to `true`.
-- **client_remove_interval** (*Optional*): How long a client is to be unavailable for before it is cleaned up. Defaults to `600 seconds (10min)`.
+{% configuration %}
+entity_namespace:
+  description: Prefix for entity ID's. Useful when using overlapping components (ex. Apple TV and Plex components when you have Apple TV's you use as Plex clients). Go from _media_player.playroom2_ to _media_player.plex_playroom_
+  required: false
+  default: `null`
```
Comment (WofWca) [inline source/_components/media_player.plex.markdown]: The problem's with ``` ` ```s.

Comment (oncleben31) [reply source/_components/media_player.plex.markdown]: thanks but build still fail

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @oncleben31!

Happy Hacktoberfest! 🎉 

../Frenck

**Pull request merged**